src/network/suppress_message.h

```diff
@@ -0,0 +1,63 @@
+#ifndef INCLUDED_NETWORK_SUPPRESS_H
+#define INCLUDED_NETWORK_SUPPRESS_H
+
+#include "network/message.h"
+#include "network/message_handler_sub_system.h"
+#include "network/message_sender_system.h"
+#include "platform/export.h"
+#include "engine/system_suppressor.h"
+
+namespace network {
+
+class SuppressMessage : public Message
+{
+    friend class ::boost::serialization::access;
+public:
+    DEFINE_MESSAGE_BASE(SuppressMessage)
+    engine::SystemSuppressor::Type mState;
```

Comment:
please specify the underlying type at the enum definition, otherwise this will break crossplatform multiplayer